src/fides/core/user.py

```diff
@@ -64,9 +64,9 @@ def create_user(
 
 def get_user_permissions(
     user_id: str, auth_header: Dict[str, str], server_url: str
-) -> List[str]:
+) -> Tuple[List[str], List[str]]:
```

Comment:
Since scopes in the database are nullable now I think the database could also return `None` for these now?

Reply:
Thanks for pointing out this case.  We do set a default code-side, but you're right this could have returned null if it was set some other way.

Thinking more I'd really like the defaults for these to be empty lists, not null, so I've changed these back to nullable being false and added a server default https://github.com/ethyca/fides/pull/2671/files#diff-a1d112d2dcc021ea7afc1ab1d87855a1d12c47c33c42f28299cef315bb71c8b9R29-R43